V1.1 (#24)
balena-io/balena-fin: documentation/markdown/balenaFin/v1.1/datasheet/datasheet.md

<center>

![Fin logo](./pictures/fin_logo.png)

| **Document Type** | Datasheet |
| --- | --- |
| **Product ID:** | BLNFN100001 (20173009) |
| **Product Name** | BalenaFin |
| **Product Version** | 1.1.0 |
| **Manufacturer Part Number** | FIN10001-X (where X is the storage size: 8/16/32/64) |
| **Document Version** | 0.0.1 |
| **Author** | Carlo Maria Curinga |
| **State (Draft/Proposed/Approved)** | Draft |

</center>

## Revision history

<center>

| **Date (dd/mm/yyyy)** | **Version** | **Author** | **Description** |
| --- | --- | --- | --- |
| 13/02/2019 | 0.0.1 | Carlo Maria Curinga | First draft |

</center>

<div class="page-break"></div>

# 1. Introduction

BalenaFin is a carrier board for the Raspberry Pi Compute Module 3 Lite and Compute Module 3+ Lite produced by the Raspberry Pi Foundation.

> For more informations about the Raspberry Pi Compute Module please refer to the following links:

Reviewer: ```suggestion
For more information about the Raspberry Pi Compute Module please refer to the following links:
```

* Datasheet: https://www.raspberrypi.org/documentation/hardware/computemodule/datasheet.md
* Schematics: https://www.raspberrypi.org/documentation/hardware/computemodule/schematics.md

<div class="page-break"></div>

# 2. Availability and Support

> BalenaFin (current version or a compatible later revision) availability is guaranteed until at least to January 2024

Reviewer: ```suggestion
Availability of balenaFin in either the current version or a compatible later revision is guaranteed until at least January 2024.
```

| 3 | ACT Status LED | CM3L Activity LED. This is the same as the green LED on the Raspberry Pi 3 Model B |
| 4 | SPD Status LED | Ethernet Speed LED. off when in 10-Mbps mode, on when in 100-Mbps mode |
| 5 | FDX Status LED | Ethernet Full-Duplex indicator |
| 6 | LNK Status LED | Ethernet Link/Activity LED. |
| 7 | PAN Status LED | If supported by the mPCIE (32) card connected, indicates PAN network activity |
| 8 | LAN Status LED | If supported by the mPCIE (32) card connected, indicates LAN network activity |
| 9 | WAN Status LED | If supported by the mPCIE (32) card connected, indicates WAN network activity |
| 10 | DSI/CAM1 connector | Standard full-size Raspberry Pi MIPI connector that can be configured as Display or secondary Camera (cam1) connector. Selection is made via #37 |
| 11 | HDMI | Full-size HDMI Type A with CEC support |
| 12 | CSI connector | Standard full-size Raspberry Pi Camera (cam0) connector |
| 13 | HAT connector | 40-pin Raspberry Pi HAT (Hardware Attached on Top) standard connector |
| 14 | WiFi/BT combo chip | 802.11ac/a/b/g/n 2.4 &amp; 5GHz WiFi + Bluetooth 4.2 |

> | 15 | WiFi/BT uFL antenna connector | If the RF switch is set on the external position, the antenna attached to this connector will become the main Radio antenna for the WiFi/BT combo chip (14) |

Reviewer: ```suggestion
| 15 | WiFi/BT uFL antenna connector | If the RF switch (33) is set on the external position, the antenna attached to this connector will become the main radio antenna for the WiFi/BT combo chip (14) |
```

| 33 | Antenna switch | 2 position switch - when set to OFF (labeled in silkscreen as "INT"), the WiFi/BT combo chip (14) uses the WiFi/BT embedded antenna (16). When set to ON (labeled in silkscreen as "EXT"), the WiFi/BT combo chip (14) uses the WiFi/BT uFL antenna connector (15) |
| 34 | PoE HAT headers | exposes the incoming voltage from the RJ45 (21) port for PoE HATs that step down and flow 5V to the 5V HAT (13) pins |
| 35 | USB 2.0 4-pin header | Exposes a USB 2.0 port via male headers. pin1 is marked in silkscreen with a triangle |
| 36 | GND probe interface | Exposes a GND probe interface for easy debugging |
| 37 | DSI/CAM1 switch | Switches the full-size Raspberry Pi MIPI connector (10) between Display or secondary Camera (cam1) mode - when set to OFF (labeled in silkscreen as "DISP"), the full-size Raspberry Pi MIPI connector (#10) exposes the DSI (disp1) interface. When set to ON (labeled in silkscreen as "CAM1") the full-size Raspberry Pi MIPI connector (10) exposes the secondary CSI (cam1) interface |
| 38 | POWER IN Fuse (on 23 & 24) | 3A 125VAC/VDC fuse - MPN: 0154003.DR |
| 39 | HAT 5V Fuse | 3A 125VAC/VDC fuse - MPN: 0154003.DR |

> 

Reviewer: ```suggestion
<div class="page-break"></div>

```
